great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -339,17 +391,57 @@ def _get_auxiliary_profiler_execution_details(self, verbose: bool) -> dict:
 
         return auxiliary_info
 
+    def _get_expectation_suite_without_usage_statistics(
```

Comment:
Is this identical to `_get_expectation_suite_with_usage_statistics` except for the decorator? If so could you change that function to call this one?
```
@usage_statistics_enabled_method(
        event_name=UsageStatsEvents.DATA_ASSISTANT_RESULT_GET_EXPECTATION_SUITE.value,
        args_payload_fn=get_expectation_suite_usage_statistics,
    )
    def _get_expectation_suite_with_usage_statistics(
        self,
        expectation_suite_name: str,
        include_profiler_config: bool = False,
    ) -> ExpectationSuite:
        return self._get_expectation_suite_without_usage_statistics(...)
```

Reply:
@billdirks I see the idea, but once there is a decorator, the event would be sent.  Did you mean to express it differently?  Thanks.

Reply:
I also like this idea. I think it's an attempt to reduce code duplication, with the "bulk" of the logic living in the `without_usage_statistics()` method

Reply:
Yeah, that wasn't clear. Let me see if I can explain what I mean better. You'd still have 2 methods and they'd look like this:
```
def _get_expectation_suite_without_usage_statistics(
    self, 
    expectation_suite_name: str, 
    include_profiler_config: bool = False, 
) -> ExpectationSuite:
    # Code logic goes here

@usage_statistics_enabled_method( 
    event_name=UsageStatsEvents.DATA_ASSISTANT_RESULT_GET_EXPECTATION_SUITE.value, 
    args_payload_fn=get_expectation_suite_usage_statistics, 
)
def _get_expectation_suite_with_usage_statistics(
    self, 
    expectation_suite_name: str, 
    include_profiler_config: bool = False, 
) -> ExpectationSuite:
    return self._get_expectation_suite_without_usage_statistics(
        expectation_suite_name, include_profiler_config)
```

Now, I can call `self._get_expectation_suite_without_usage_statistics()` and no usage metrics will be sent since this method is undecorated or I can call `self._get_expectation_suite_with_usage_statistics()` to send metrics.

Reply:
❤️ @billdirks @Shinnnyshinshin one second, please... :)